### DISCUSSION TITLE
RTSP to the screen and to a server at the same time

### DISCUSSION THREAD

**Discussion by jjcetraro** [pedroSG94/RootEncoder#1000] [General]
Hi! I need to take an RTSP stream located in some host (example: rtsp:\\192.168.1.14:8554\mystream), and show it on the screen of the android device, and at the same time I have to send this stream to a server. Can it be done with this library?

Thanks!

**Comment by pedroSG94** [comment]
Hello,

It is not possible, reproduce a stream is not supported in this library.
In this case you need to find the way to reproduce the stream and show it in your device using other library. After that, you can use rtsp module to send raw H264 and raw AAC frames to server.
